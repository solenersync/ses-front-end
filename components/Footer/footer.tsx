import { NextComponentType } from "next";

const Footer: NextComponentType = () => {
  return (
    <footer className="mt-24 sm:mt-6 text-center flex justify-center">
      <div className="mx-auto max-w-md overflow-hidden py-12 px-6 sm:max-w-3xl lg:max-w-7xl lg:px-8">
        <div className="mt-8 flex justify-center space-x-6">
          <a
            href="https://www.solenersync.net/dashboard"
            rel="noopener noreferrer"
          >
            <img
              className="md:h-20 h-16 w-auto"
              src="logo-small.png"
              alt="Solenersync"
            />
          </a>
        </div>
        <p className="mt-8 text-center text-base text-gray-400">
          &copy; 2022 Solenersync. All rights reserved.
        </p>
      </div>
    </footer>
  );
};
export default Footer;
