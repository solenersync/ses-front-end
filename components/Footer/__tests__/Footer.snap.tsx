import { render } from "@testing-library/react";
import Footer from "components/Footer/footer";

describe("Footer", () => {
  test("should render the footer", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
