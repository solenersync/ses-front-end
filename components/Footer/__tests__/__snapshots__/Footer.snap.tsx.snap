// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the footer 1`] = `
<div>
  <footer
    class="mt-24 sm:mt-6 text-center flex justify-center"
  >
    <div
      class="mx-auto max-w-md overflow-hidden py-12 px-6 sm:max-w-3xl lg:max-w-7xl lg:px-8"
    >
      <div
        class="mt-8 flex justify-center space-x-6"
      >
        <a
          href="https://www.solenersync.net/dashboard"
          rel="noopener noreferrer"
        >
          <img
            alt="Solenersync"
            class="md:h-20 h-16 w-auto"
            src="logo-small.png"
          />
        </a>
      </div>
      <p
        class="mt-8 text-center text-base text-gray-400"
      >
        Â© 2022 Solenersync. All rights reserved.
      </p>
    </div>
  </footer>
</div>
`;
